ocpp/routing.py

```diff
@@ -32,7 +33,10 @@ def inner(*args, **kwargs):
 
         inner._on_action = action
         inner._skip_schema_validation = skip_schema_validation
+        if func.__name__ not in routables:
+            routables.append(func.__name__)
```

Comment:
The last registred route will override the earlier once.

Reply:
only by name, so it does not matter. When building the route_map for the individual object it works out fine. Added a test case to make sure of that 

def test_getters_should_not_be_called_during_routemap_setup():
    class ChargePoint(cp):
        @property
        def foo(self):
            raise RuntimeError("this will be raised")

    try:
        ChargePoint("blah", None)
    except RuntimeError as e:
        assert str(e) == "this will be raised"
        pytest.fail("Getter was called during ChargePoint creation")

def test_multiple_classes_with_same_name_for_handler():
    class ChargerA(cp):
        @on(Action.Heartbeat)
        def heartbeat(self, **kwargs):
            pass

    class ChargerB(cp):
        @on(Action.Heartbeat)
        def heartbeat(self, **kwargs):
            pass
            
    A = ChargerA("A", None)
    B = ChargerB("B", None)
    route_mapA = create_route_map(A)
    route_mapB = create_route_map(B)
    assert route_mapA["Heartbeat"] != route_mapB["Heartbeat"]

Reply:
Of course, I was wrong when stating that this wouldn't work.